diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/LineLength.kt

```diff
@@ -209,6 +217,12 @@ class LineLength : Rule("line-length") {
         return null
     }
 
+    /**
+     * This method stored all the nodes that have BINARY_EXPRESSION element type.
```

Comment:
What about `PREFIX_EXPRESSION` that you add on the last iteration? Why is it needed?